packages/system/src/background.ts

```diff
@@ -10,57 +10,77 @@ type AddProperty<T, T2> = {
 // eslint-disable-next-line @typescript-eslint/ban-types
 export type BackgroundProps<AutoPrefix extends string = string & {}> = Partial<
   AddProperty<
-    {
-      /**
-       * @see backgroundImage
-       */
-      bgImage: CSSValue<"backgroundImage"> | AutoPrefix;
-      /**
-       * @see backgroundPosition
-       */
-      bgPosition: CSSValue<"backgroundPosition"> | AutoPrefix;
-      /**
-       * @see backgroundPositionX
-       */
-      bgPositionX: CSSValue<"backgroundPositionX", true> | AutoPrefix;
-      /**
-       * @see backgroundPositionY
-       */
-      bgPositionY: CSSValue<"backgroundPositionY", true> | AutoPrefix;
-      /**
-       * @see backgroundSize
-       */
-      bgSize: CSSValue<"backgroundSize", true> | AutoPrefix;
-      /**
-       * @see backgroundRepeat
-       */
-      bgRepeat: CSSValue<"backgroundRepeat"> | AutoPrefix;
-      /**
-       * @see backgroundAttachment
-       */
-      bgAttachment: CSSValue<"backgroundAttachment"> | AutoPrefix;
-      /**
-       * @see backgroundClip
-       */
-      bgClip: CSSValue<"backgroundClip"> | AutoPrefix;
-      /**
-       * @see backgroundOrigin
-       */
-      bgOrigin: CSSValue<"backgroundOrigin"> | AutoPrefix;
-    },
+    CSSProperties<
+      | "backgroundImage"
+      | "backgroundPosition"
+      | "backgroundRepeat"
+      | "backgroundAttachment"
+      | "backgroundClip"
+      | "backgroundOrigin"
+    > &
+      CSSProperties<
+        "backgroundPositionX" | "backgroundPositionY" | "backgroundSize",
+        true
+      > & {
+        /**
+         * @see backgroundImage
+         */
+        bgImage: CSSValue<"backgroundImage"> | AutoPrefix;
```

Comment:
Those `AutoPrefix` looks needless since it's in `AddProperty` but keeping it left as it's out of scope of this PR.

Reply:
Oh thanks. Honestly I don't know when and where this came from lol